css/loginStyles.css

```diff
@@ -0,0 +1,69 @@
+body {
+    background-color: #406bb7;
+}
+
+.bg {
+    width: 95%;
+    background-color: #ffffff;
+    border: solid 5px #406bb7;
+    border-radius: 12px;
+    margin: 0 auto;
+}
+
+.title, .title2 {
+    background-image: url("../images/pokebees.jpg");
+    background-size: cover;
+    margin: 0 auto;
+    width: 600px;
+    height: 180px;
+
+}
+
+.pokeball {
+    position: relative;
+    background-image: url("../images/pokeball.png");
+    background-size: cover;
+    margin: 0 auto;
+    width: 400px;
+    height: 400px;
+
+}
+
+.title2 {
+    background-image: url("../images/pokedex.jpg");
+    height: 210px;
+}
+
+.login {
+    position: absolute;
+    top: 40%;
+    left: 50%;
+    transform: translate(-50%);
+    width: 150px;
+    height: 100px;
+    margin: 0 auto;
```

Comment:
Looks like you are using two methods to center at the same time. get rid of this one. It works without

Reply:
removed